cmd/notation-azure-kv/metadata.go

```diff
@@ -41,8 +25,7 @@ func init() {
 	}
 }
 
-func runGetMetadata(ctx *cli.Context) error {
+func runGetMetadata() {
```

Comment:
We can keep returning the `error`. Note that `os.Stdout.Write()` returns an error.

Reply:
Updated.